assets/js/vendor.js

```diff
@@ -13770,8 +13770,9 @@ return VueRouter;
 		} // end _controls
 
 	});
-
 	$.fn.extend({
+    // Adding custom jQuery serialize wrapper to the frontend and backend for WeForms 1.6.7. 
+    // GitHub ticket created to make one script applied to frontend/backend.

```

Comment:
Instead of an additional comment block above, how about this:

```php
       /**
		* Custom jQuery serialize wrapper.
		*
		* When WordPress 5.6 increased the jQuery version to 3.5.1, the serialize function changed. Instead of
		* sending spaces as "+", they are sent as "%20". This wrapper is for backwards compatibility.
		* 
		* @todo This function is duplicated in both the frontend and backend. Need to have the code live in
		* just one location.
		*
		* @since ###
		*/
```

Reply:
removed blocks and added to the block with the `@todo` tag.